aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -51,7 +51,7 @@ The `Startup` class has the following additions:
         options.UseSqlite(Configuration.GetConnectionString("DefaultConnection")));
 
     services.AddDefaultIdentity<ApplicationUser>()
-        .AddDefaultUI(UIFramework.Bootstrap4)
+        .AddDefaultUI()
```

Comment:
This line can be removed completely; `AddDefaultIdentity` already includes it.

Reply:
You are right, I removed the line.